odoo/fields.py

```diff
@@ -4826,10 +4827,12 @@ def relation_delete(ys):
         pairs = [(x, y) for x, ys in new_relation.items() for y in ys - old_relation[x]]
         if pairs:
             if self.store:
-                query = "INSERT INTO {} ({}, {}) VALUES {} ON CONFLICT DO NOTHING".format(
-                    self.relation, self.column1, self.column2, ", ".join(["%s"] * len(pairs)),
+                query = SQL("INSERT INTO {} ({}, {}) VALUES %s ON CONFLICT DO NOTHING").format(
+                    Identifier(self.relation),
+                    Identifier(self.column1),
+                    Identifier(self.column2),
                 )
-                cr.execute(query, pairs)
+                execute_values(cr._obj, query, pairs)
```

Comment:
Hi, @flvr-odoo I have a small question about this `execute_values`.
Since it will call `cr._obj.execute` directly which bypasses all override logics in the `cr.execute`. As a result,
1. --log-sql cannot log these queries
2. assertQueryCount cannot notice these queries
...

Is this an expected/acceptable behavior?